roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -194,6 +220,33 @@
     dest: "{{ cifmw_libvirt_manager_basedir }}/artifacts/interfaces-info.yml"
     content: "{{ cifmw_libvirt_manager_mac_map | to_nice_yaml }}"
 
+- name: Refresh and dump vbmc hosts
+  when:
+    - _vbmc_host is defined
+    - _vbmc_host == inventory_hostname
+  block:
+    - name: Ensure fresh vbmc listing
+      ansible.builtin.include_role:
+        name: virtualbmc
+        tasks_from: list_hosts.yml
+
+    - name: Dump vbmc known hosts
+      vars:
+        _auth:
+          username: "{{ cifmw_virtualbmc_ipmi_user | default('admin') }}"
+          password: "{{ cifmw_virtualbmc_ipmi_password | default('password') }}"
+        _mapped: >-
+          {{
+            cifmw_virtualbmc_known_hosts |
+            map('combine', _auth)
+          }}
+        content:
+          cifmw_virtualbmc_known_hosts: "{{ _mapped }}"
+      ansible.builtin.copy:
+        dest: >-
+          {{ cifmw_libvirt_manager_basedir }}/artifacts/virtualbmc-nodes.yml
+        content: "{{ content | to_nice_yaml }}"
```

Comment:
a proper `mode` should be declared.

Reply:
Right - after checking, there are some other missing `mode` in that same file. I propose to merge as-is, and get someone to converge the missing `mode` in the whole repository in follow-ups.